imports/ui/pages/landingPage.html

```diff
@@ -11,11 +11,6 @@
                     <a href="/sign-up" class="button button-large button-red">Join</a>
                     <a href="/log-in" class="button button-large button-blue">Login</a>
                 </div>
-                {{#if landingPageModel}}
-                <div class="col-6">
-                    <iframe width="350" height="250" src="/models/{{landingPageModel}}/shared=true" frameborder="0"></iframe>
-                </div>
-                {{/if}}
```

Comment:
We need this model on the landing page. Do not remove it.

Currently, you used the error template for 404-page error. But this template is also being used to show when model is not displayed. So we will get this  404 error, when there is a problem in the model rendering. In the screenshot below, you can see how we got this on home page as well.
![Screenshot from 2019-06-27 17-07-09](https://user-images.githubusercontent.com/1990932/60308533-0c7acb80-98fe-11e9-9607-f95025313b2f.png)

Reply:
I would like to have a more in-depth discussion about this, because that model has not loaded and will not load on the landing page properly this whole time so I removed it.

The 404 page works correctly in other instances.. I believe the real problem is separating the model loading error from the 404 error, which is the same at the moment, that may require a separate PR to separate that, but for now I believe this works for the 404. 

I just have to fix the model loading in a separate PR. Panda please let me move on to fixing that because the 404 page works as of now, I just have to fix the model loading error separately, their are a lot of other errors regarding model loading such as scale of models and error notification as well.

Reply:
<iframe width="350" height="250" src="/models/{{landingPageModel}}/shared=true" frameborder="0"></iframe>

Is not properly calling that model so I temporarily got rid of it until I get a proper solution for loading that model. I think fixing that should be in a seperate PR and issue solving for the landing page model loading issues.